container_test.go

```diff
@@ -489,6 +490,53 @@ func TestShouldStartContainersInParallel(t *testing.T) {
 	}
 }
 
+func TestLabelBuilding(t *testing.T) {
+	labels := core.DefaultLabels("session-id")
+	assert.NotNil(t, labels)
```

Comment:
nit: I would recommend using `require` exclusively to avoid noise when one condition fails.